src/gui/systray.cpp

```diff
@@ -285,6 +285,42 @@ void Systray::destroyEditFileLocallyLoadingDialog()
     _editFileLocallyLoadingDialog = nullptr;
 }
 
+void Systray::createResolveConflictsDialog(const OCC::ActivityList &allConflicts)
+{
+    const auto conflictsDialog = new QQmlComponent(_trayEngine, QStringLiteral("qrc:/qml/src/gui/ResolveConflictsDialog.qml"));
+    const QVariantMap initialProperties{
+                                        {"allConflicts", QVariant::fromValue(allConflicts)},
+    };
+
+    if(conflictsDialog->isError()) {
+        qCWarning(lcSystray) << conflictsDialog->errorString();
+        delete conflictsDialog;
+        return;
+    }
+
+    // This call dialog gets deallocated on close conditions
+    // by a call from the QML side to the destroyDialog slot
+    auto dialog = conflictsDialog->createWithInitialProperties(initialProperties);
+    if (!dialog) {
+        delete dialog;
```

Comment:
@mgallien Are we deleting a `null` pointer?

Reply:
obviously not
good catch